DataManagementSystem/DB/FileCatalogComponents/DirectorySimpleTree.py

```diff
@@ -151,10 +147,10 @@ def getChildren(self,path):
       dirID = path
         
     req = "SELECD DirID FROM FC_DirectoryTree WHERE Parent=%d" % dirID
-    result = self._query(req)
+    result = self.db._query( req )
```

Comment:
In fact, this manager is not even used to my knowledge. We should remove all the useless managers. One could try it with the unit tests, it is designed so that you can choose your managers....